packages/core/src/indexing/postgres/__tests__/postgres-index-api.test.ts

```diff
@@ -166,6 +167,7 @@ describe('indexStream', () => {
       ...STREAM_CONTENT,
       updatedAt: updateTime,
       lastAnchor: updateTime,
+      firstAnchor: null,
```

Comment:
Do you know if there is a follow up ticket that actually sets the `firstAnchor` value?

Reply:
Just checked and there is not - I can create one and do a follow up to this one or just add it to the current PR. I saw that it's currently also not set for sqlite.

Reply:
> Just checked and there is not - I can create one and do a follow up to this one or just add it to the current PR. I saw that it's currently also not set for sqlite.

Indeed. IMO, it should be a different slice.